Demo/Demo/Networking/DemoMerchantAPI.swift

```diff
@@ -9,6 +9,7 @@ final class DemoMerchantAPI {
 
     static let sharedService = DemoMerchantAPI()
     var accessToken: String?
```

Comment:
🧶  We should be able to remove this property now!

Reply:
addressed in 2813be8fab3d5b7d0f2c0c6787ef7ce7d0c141c1